index.html

```diff
@@ -29,10 +29,10 @@
 
 
         <section class="Row-2">
```

Comment:
instead of it being your own custom class of `Row-2` try the original `.row` instead. Example: `<section class="row">`